.env

```diff
@@ -1,10 +1,17 @@
-NEXT_PUBLIC_WEBSITE_URL = "http://localhost:3000"
-NEXT_PUBLIC_LOCALHOST = "localhost"
-NEXT_PUBLIC_MAIN_LANGUAGE = "uk"
-NEXT_PUBLIC_FALLBACK_LANGUAGE = "en"
-NEXT_PUBLIC_GRAPHQL_API_URL = "https://localhost/api/graphql"
-NEXT_PUBLIC_API_URL = "https://yourserver.io/api/"
-NEXT_PUBLIC_VILNACRM_GMAIL = "info@vilnacrm.com"
-NEXT_PUBLIC_VILNACRM_PRIVACY_POLICY_URL = "https://github.com/VilnaCRM-Org"
-NEXT_PUBLIC_CONTINUOUS_DEPLOYMENT_HEADER_NAME = "no-aws-header-name"
-NEXT_PUBLIC_CONTINUOUS_DEPLOYMENT_HEADER_VALUE = "no-aws-header-value"
+NEXT_PUBLIC_PROD_CONTAINER_API_URL="http://prod:3001"
+NEXT_PUBLIC_DEV_HOST="localhost"
```

Comment:
It will be better do not duplicate the localhost
Please use one environment instead of two

```
NEXT_PUBLIC_DEV_HOST="localhost"
NEXT_PUBLIC_DEVELOPMENT_API_URL="http://localhost:3000"
```